### ISSUE TITLE
Authentication SubKey shows AR

### ISSUE THREAD

**Issue by BosnaZmaj** [drduh/YubiKey-Guide#400]
Has anyone every come across where the Authentication sub key shows AR instead of A? I wasn't very successful on google regarding this. How would one fix this? I am on  MacOS

**Comment by MalteT** [comment]
I had the same issue leading to problems in OpenKeychain on Android. I'm not sure what exactly is the issue here and couldn't find any documentaion either. During my search, I found someone complaining that the 'AR' type appeared after updating gnupg. I did some testing and found, that with gnupg LTS version 2.2 (packaged in nixpkgs as gnupg22) subkey generation worked as expected. I generated a new authentication subkey using that gnupg version and used a more recent version for the rest of the tutorial, which worked great!

TL;DR: Use gnupg version 2.2.41 for authentication subkey generation
